banzai/context.py

```diff
@@ -16,11 +16,11 @@ def __eq__(self, other):
 
 
 class PipelineContext(object):
-    def __init__(self, command_line_args, allowed_instrument_criteria, processed_path='/archive/engineering/',
-                 post_to_archive=False, fpack=True, rlevel=91, db_address='mysql://cmccully:password@localhost/test',
-                 log_level='INFO', preview_mode=False, max_tries=5, post_to_elasticsearch=False,
-                 elasticsearch_url='http://elasticsearch.lco.gtn:9200', elasticsearch_doc_type='qc',
-                 elasticsearch_qc_index='banzai_qc', **kwargs):
+    def __init__(self, command_line_args, allowed_instrument_criteria, image_class,
```

Comment:
It would be good to parameterize this with environment variables at some point.

Reply:
All of these parameters can be set from the command line currently. Is there justification to do environment variables as well?

Reply:
Nope, that's perfect.